### PR TITLE
Render json schemas in docs

### PR DESCRIPTION
**Pull request by Callisto13**
Part of https://github.com/weaveworks/profiles/issues/239, last bit before adding some styling.

Thanks to @AlinaGoaga ✨

### CODE REVIEW

File: userdocs/profiles.dev/package.json [hunk 1]
```diff
@@ -17,11 +17,13 @@
     "@docusaurus/core": "2.0.0-beta.4",
     "@docusaurus/preset-classic": "2.0.0-beta.4",
     "@mdx-js/react": "^1.6.21",
+    "@next-tools/dynamic": "^0.0.11",
```
Comment (Himangini) [inline userdocs/profiles.dev/package.json:20]: this is no longer required

File: userdocs/profiles.dev/package.json [hunk 2]
```diff
@@ -17,11 +17,13 @@
...
     "@svgr/webpack": "^5.5.0",
     "clsx": "^1.1.1",
     "file-loader": "^6.2.0",
     "prism-react-renderer": "^1.2.1",
     "react": "^17.0.1",
+    "react-code-blocks": "0.0.9-0",
```
Comment (Himangini) [inline userdocs/profiles.dev/package.json:26]: same here

File: userdocs/profiles.dev/src/components/Schema.js [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Modifications:
+  Copyright 2021 Weaveworks
+*/
+
+import React, { useState, useEffect } from "react";
+
+const Schema = (input) => {
```
Comment (Himangini) [inline userdocs/profiles.dev/src/components/Schema.js:22]:
```suggestion
const Schema = ({jsonFile}) => {
```
you can use the prop directly

**Review by Himangini — COMMENTED** [review]
there's a warning on browser 
```
Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
```
might need a useEffect cleanup function, but otherwise everything looks good 👍🏻

**Comment by Himangini** [comment]
minor thing, the formatting here looks a bit off for code blocks 🤷🏻‍♀️  
Screenshot 2021-08-12 at 13 46 01

**Comment by Callisto13** [comment]
ah interesting! those are backticks in the go. will fix

**Comment by Callisto13** [comment]
Console error gone, and rendering "fixed" (as in I did the cheapest thing to get this closed for now, and we can deal with it in a separate styling ticket)

**Comment by Himangini** [comment]
> Console error gone, and code block rendering "fixed" (as in I did the cheapest thing to get this closed for now, and we can deal with it in a separate styling ticket)

cool. yeah, we can deal with those `<code>` blocks in markdown later.

**Review by Himangini — APPROVED** [review]
👌🏻

**Pull request merged**